docs/features/exportpdf/README.md

```diff
@@ -0,0 +1,106 @@
+
+## Supported features
+
+All the formatting and content structure you create using the editor is maintained in the output file. See the tips for {@link features/exportpdf/README#reaching-the-best-projection reaching the best projection} to learn how to configure the plugin and editor to best suite your needs.
+
+As for the compatibility, plugin works on all the browsers {@link guide/dev/browsers/README supported for CKEditor 4} except for Internet Explorers older than version 11.
+
+## Configuration
+
+### Reaching the best projection
+
+Due to the differences between browsers and operating systems it is not always possible to reach a perfect match between content in the editor and PDF. However, thanks to flexible configuration, adjusting couple of configuration options can make the difference hardly noticeable.
+
+Make the editor width correspond with the chosen paper format for output file - e.g. for the `A4` format (which is used by the export service [by default](https://pdf-converter.cke-cs.com/docs#section/PDF-options/Page-format)) the editor width should be equal to `840px`. Then mind the margins - if they are changed for the PDF using [custom config](https://pdf-converter.cke-cs.com/docs#section/PDF-options/Margins), also update them for editor using {@linkapi CKEDITOR.addCss} method. To take things a step further you may experiment with {@link guide/dev/example_setups/README#document-editor Document Editor setup}.
+
+### Custom CSS rules
+
+Besides the inline styles, in order to mimic the editor content, all the CKEditor 4 CSS rules are sent to the service. But let's say your company has a strict heading color policy: it has to be <span style="color:#4B0082">indigo</span> (`#4B0082`), definitely not <span style="color:#483D8B">dark slate blue</span> (`#483D8B`)! Fortunately you do not have to remember to change it every time. It is possible to attach your own custom CSS stylesheet via {@linkapi CKEDITOR.config.exportPdf_stylesheet} configuration option or just add some styles with {@linkapi CKEDITOR.addCss} method.
+
+### Output file configuration
+
+A number of options like output file format or margins can be set in the {@linkapi CKEDITOR.config.exportPdf_options} object. It is sent to the service along with the HTML and CSS and processed on the server side. To check out the possible configurations, visit the [service documentation](https://pdf-converter.cke-cs.com/docs).
+
+### Data preprocessing
+
+Plugin provides a custom {@linkapi CKEDITOR.editor#exportPdf exportPdf event}. It can be used for custom data processing (e.g. to ensure the output text will be black, not <span style="color:pink;background-color:yellow">pink on yellow background</span>). Editor uses it too, so just remember to add the right priority to the listener:
```

Comment:
I also linked to a community plugin: https://ckeditor.com/cke4/addon/base64image.

Reply:
That's a good idea, however... Basically, by mentioning community plugin we recommend it so we should make sure everything works fine there. Also there are [more base64 image plugins](https://ckeditor.com/cke4/addons/search/plugins/base64) so we should link to the most reliable one :thinking: Or we could link to list of all image base64 plugins (e.g. `https://ckeditor.com/cke4/addons/search/plugins/base64`).

Since we have issue for converting images to base64 representation already reported I'm not sure if this mention is needed?